projects/core/styles/components/appearance.less

```diff
@@ -26,7 +26,7 @@
     appearance: none;
     outline: 0.125rem solid transparent;
     outline-offset: -0.125rem;
-    transition-property: color, background, opacity, box-shadow, border, border-radius;
+    transition-property: color, background-color, opacity, box-shadow, border, border-radius;
```

Comment:
Otherwise, `background-image` is animated too


https://github.com/user-attachments/assets/2034300d-ad99-494c-90b3-9084a95a5c3c